README.md

```diff
@@ -26,14 +26,14 @@
 </a>
 
 <h1 align="center">
-  Welcome to Kodi Home Theater Software!
+  Welcome to Kodi Home Theatre Software!
```

Comment:
In AE (American English) the spelling is "Theater". In BE (Britain English) the Spelling is "Theatre". So there's nothing wrong while using "Theater", especially as Kodis trademark is a US trademark.